### DISCUSSION TITLE
"ImageReward is not installed, cannot be used"

### DISCUSSION THREAD

**Discussion by Kapowpenguin1** [AlUlkesh/stable-diffusion-webui-images-browser#177] [Q&A] [answered]
what is this "imagereward", how do i install it, and how do i use it?

**Comment by halr9000** [comment]
I found this, but have not got it fully working quite yet. 

https://github.com/THUDM/ImageReward#integration-into-stable-diffusion-web-ui

what I've done:

1. from terminal, venv/scripts/activate.ps1
2. run pip install command from above URL
3. I copied the [demo script](https://github.com/THUDM/ImageReward/blob/main/demo/sdwebui/image_reward.py )to my scripts/ folder, but unsure if that's needed for the image browser.

The error has gone away, but I'm unable to generate scores inside the image browser.

**Comment by AlUlkesh** [comment, marked as answer]
Please read this: #146

**Reply by Kapowpenguin1** [reply]
sounds like it's just not working properly rn
